.travis.yml

```diff
@@ -10,23 +10,29 @@ notifications:
 
 jdk:
   - oraclejdk8
-  - oraclejdk7
 
 android:
   components:
     - platform-tools
     - tools
-    - build-tools-23.0.3
-    - android-23
+    - build-tools-24.0.3
+    - android-24
     - extra-google-m2repository
     - extra-android-m2repository
-    - sys-img-armeabi-v7a-android-23
+    - sys-img-armeabi-v7a-android-24
```

Comment:
I forgot to say that in you Auth PR. You should remove the sys image from here since we are installing it from matrix. It prevent to install on all matrix tests